src/cljs/owlet_ui/components/activity/comments.cljs

```diff
@@ -0,0 +1,15 @@
+(ns owlet-ui.components.activity.comments
+  (:require [reagent.core :as reagent]
+            [re-frame.core :as re]))
+
+(defn load-comments [activity]
+  (js/disqusReset
+    (get-in activity [:sys :id])
+    (clojure.string/replace (-> js/document .-location .-href) "#" "#!")
+    (get-in activity [:fields :title])))
+
+(defn activity-comments [activity-info]
```

Comment:
Thanks @cjford this is great progress!
Can we get rid of this function parameter `activity-info` since its not being used in the function ?